QUEST_LV_0200.tsv

```diff
@@ -7739,87 +7739,87 @@ QUEST_LV_0200_20150714_007742	I should've persuaded him slowly, but I think we w
-QUEST_LV_0200_20150714_007753	This place is dangerous.{nl}Be careful all the time.
-QUEST_LV_0200_20150714_007754	We've passed the dangerous situation with an emergency procedure, but...{nl}If we don't detoxify it completely, there will be more crisis.{nl}
+QUEST_LV_0200_20150714_007753	This place is dangerous.{nl}Be cautious at all times.
+QUEST_LV_0200_20150714_007754	We've passed the dangerous situation with an emergency procedure, but...{nl}If we don't detoxify it completely, there will be more crises.{nl}
 QUEST_LV_0200_20150714_007755	But, I saw this poison for the first time.{nl}While I am taking care of him, please get me the other poison so I could compare.
 QUEST_LV_0200_20150714_007756	What we should obtain is the poison from thorns of Chafferors
 QUEST_LV_0200_20150714_007757	The poison must have been made using stuff that can be found around this area. {nl}Please hurry before it's too late.
 QUEST_LV_0200_20150714_007758	It is so fortunate that we discovered it before it's too late.
-QUEST_LV_0200_20150714_007759	You are not late.{nl}I hope one of these three poisons work.
-QUEST_LV_0200_20150714_007760	We were going to go find Wugushi in the city, but fortunately it seems that we don't need to do that.{nl}I think I will be able to make an antidote if we have the leaves from Wurudwa.
-QUEST_LV_0200_20150714_007761	I was worried that we may have to move to another city.{nl}If we have to look for Wugushi there, then we can't guarantee our lives.
-QUEST_LV_0200_20150714_007762	I am going to absorb poison with this.{nl}I am going to keep looking at him so please care of Marko who is uneasy.
+QUEST_LV_0200_20150714_007759	You are not too late.{nl}I hope one of these three poisons work.
+QUEST_LV_0200_20150714_007760	We were going to go find a Wugushi in the city, but fortunately it seems that we don't need to do that.{nl}I think I will be able to make an antidote if we have the leaves from Wurudwa.
+QUEST_LV_0200_20150714_007761	I was worried that we may have to move to another city.{nl}If we have to look for a Wugushi there, then we can't guarantee our lives.
+QUEST_LV_0200_20150714_007762	I'll absorb the poison with this.{nl}I am going to keep looking over him so please care of Marko, he seems uneasy.
 QUEST_LV_0200_20150714_007763	You won't understand how embarassed and painful I am.{nl}My brother almost died due to the poisonn and I received help from the hunters that I kicked out...{nl}
 QUEST_LV_0200_20150714_007764	Good. You would have to persuade Guus at Laukyme Swampy Place.{nl}Please tell Marko that I've decided to go with Hunters.
 QUEST_LV_0200_20150714_007765	It's is so fortunate that it turned out alright.{nl}Please meet Captain Mintz on your way to Laukyme Swampy Place and tell him that everything went alright.
 QUEST_LV_0200_20150714_007766	Senior monk, Guus is leading the brothers at Laukyme Swampy Place.{nl}If Evoniphon did all of these, the Goddesses would also understand my rage.
```

Comment:
Senior monk, Guus -> Senior Monk Guus

If Evoniphon did all of these -> If Evoniphon did all of this (?)